app.rb

```diff
@@ -0,0 +1,101 @@
+require_relative 'person'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def list_books
+    Book.all.each do |book|
+      puts "Title: #{book.title}"
+      puts "Author: #{book.author}"
+      puts '----------'
+    end
+  end
+
+  def list_people
+    Person.all.each do |person|
+      puts "Name: #{person.correct_name}"
+      puts "Age: #{person.age}"
+      puts "Classroom: #{person.classroom&.label}" if person.is_a?(Student)
+      puts "Specialization: #{person.specialization}" if person.is_a?(Teacher)
+      puts '----------'
+    end
```

Comment:
Hello, @jkanyi-web ,
Good job coming this far with your project 🚀 
Please kindly fix this error in your "person.rb" file. Your code isn't running as a result of the error encountered on line 4.
Kindly fix all errors shown in your console before submitting for a review again 👍 

Console error:
![image](https://github.com/jkanyi-web/OOP_School_Library/assets/20537712/a0c131ac-331b-4209-868d-97a802ae683e)

Reply:
Addressed